src/components/dataset/data/sidebar/DataViewSidebarItem.jsx

```diff
@@ -25,13 +25,9 @@ export class DataViewSidebarItem extends React.PureComponent {
   static propTypes = {
     classes: PropTypes.object,
     column: PropTypes.object,
-    dataset: PropTypes.object,
     filterData: PropTypes.object,
-    filterStatement: PropTypes.string,
     handleChange: PropTypes.func,
-    joinStatement: PropTypes.string,
     tableName: PropTypes.string,
-    token: PropTypes.string,
```

Comment:
We no longer need to pass these items through:
- Dataset & filterStatement have been moved to redux state
- joinStatement is not needed for the column stats endpoint
- token was only used when directly talking to bq